affiliates/philippines.swagger.json

```diff
@@ -0,0 +1,490 @@
+            "required" : false,
+            "style" : "simple",
+            "explode" : false,
+            "schema" : {
+              "format" : "string"
+            },
+            "example" : "fil-PH"
+          } ],
+          "requestBody" : {
+            "description" : "Customer info",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/Customer"
+                }
+              },
+              "application/xml" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/Customer"
+                }
+              }
+            }
+          },
+          "responses" : {
+            "200" : {
+              "description" : "Customer is registered",
+              "content" : {
+                "application/json" : {
+                  "schema" : {
+                    "$ref" : "#/components/schemas/CustomerResponse"
+                  }
+                },
+                "application/xml" : {
+                  "schema" : {
+                    "$ref" : "#/components/schemas/CustomerResponse"
+                  }
+                }
+              }
+            },
+            "401" : {
+              "description" : "Unauthorized",
+              "content" : {
+                "application/json" : {
+                  "schema" : {
+                    "$ref" : "#/components/schemas/UnauthorizedResponse"
+                  }
+                },
+                "application/xml" : {
+                  "schema" : {
+                    "$ref" : "#/components/schemas/UnauthorizedResponse"
+                  }
+                }
+              }
+            },
+            "422" : {
+              "description" : "Validation error",
+              "content" : {
+                "application/json" : {
+                  "schema" : {
+                    "$ref" : "#/components/schemas/ValidationErrors"
+                  }
+                },
+                "application/xml" : {
+                  "schema" : {
+                    "$ref" : "#/components/schemas/ValidationErrors"
+                  }
+                }
+              }
+            }
+          },
+          "security" : [ {
+            "oAuth" : [ ]
+          } ]
+        }
+      },
+      "/leads" : {
+        "post" : {
+          "description" : "Register lead",
+          "parameters" : [ {
+            "name" : "Accept-Language",
+            "in" : "header",
+            "description" : "To get response error messages in filipino, header’s value must be set to according ISO code. Ex: `fil` or `fil-PH`.",
+            "required" : false,
+            "style" : "simple",
+            "explode" : false,
+            "schema" : {
+              "format" : "string"
+            },
+            "example" : "fil-PH"
```

Comment:
we actually do not support `fil-PH` language

Reply:
removed this